README.md

```diff
@@ -91,23 +112,7 @@ Then, run the following command:
 $ pod install
 ```
 
-### Swift Package
-The [Swift Package Manager](https://swift.org/package-manager/ "") is a tool for automating the distribution of Swift code and is integrated into the swift compiler.
-
-Once you have your Swift package set up, adding Alamofire as a dependency is as easy as adding it to the `dependencies` value of your `Package.swift`.
-```swift
-dependencies: [
-    .package(url: "https://github.com/web3swift-team/web3swift.git", .upToNextMajor(from: "3.0.0"))
-]
-```
-
-## Example usage
-In the imports section:
-
-```swift
-import web3swift
-import Web3Core
-```
+> **WARNING**: CocoaPods is a powerful tool for managing dependencies in iOS development, but it also has some limitations that need to be considered such as its limited control over dependencies, lack of transparency, complex setup and slow performance. We highly recommend using SPM first as using CocoaPods will delay new updates and bug fixes being delivered to you.
```

Comment:
No that i've been considered by this, but this is both not quite true about Cocoapods limitations nor why we are treat it as a second class citizen. 

Just for history: The reason why we're going towards to drop Cocoapods support is that it's having a leak of modules support. So at a contrast to SPM it's impossible to made several modules in our project and to distribute them as one package.